doc/locale/en/LC_MESSAGES/1.1_demo_imageprocessing.po

```diff
@@ -51,64 +51,64 @@ msgstr ""
 "Java Runtime Environment (JRE) 7 (Bundled at "
 "`../misc/openjdk_1.7.0_u45_2.4.3_installed`, no need to install)"
 
-#: ..\..\1.1_demo_imageprocessing.rst:29
+#: ..\..\1.1_demo_imageprocessing.rst:30
```

Comment:
Looks like these number after the colon is either line numbers or entry ID in this `.po` file. Whenever an entry is added, these numbers have to be incremented, which makes the diff a bit complicated but I don't know if there's a way to avoid this. Not a big issue.